components/loading-box/src/index.js

```diff
@@ -30,15 +30,19 @@ const Innerwrap = glamorous.div(({
   bottom: 0,
   display: 'flex',
   justifyContent: 'center',
+  alignItems: 'center',
   [`& .${spinnerClassName}`]: {
-    position: 'absolute',
     zIndex: 101,
     opacity: 1,
     display: 'block',
-    height: '100%',
     maxHeight: 'calc(50vh + 100px)',
     transition: `opacity ${timeIn}ms ease-in-out`,
   },
+  [`& .${spinnerClassName} + p`]: {
```

Comment:
not sure what all this extra css is needed for just to pass the prop down.. if it's needed could you update the pr description?